docs/src/site/optics.md

```diff
@@ -6,5 +6,12 @@ section: "optics"
 
 # Optics
 
+## Convert between two types
```

Comment:
Having access to the Prism (weaker) api in Iso would be confusing, better not use inheritance.